app/views/notifications/_original_aggregated_reactions.html.erb

```diff
@@ -0,0 +1,49 @@
+<%# TODO: change to map of IDs %>
```

Comment:
Again, I did _not_ change anything in these `original` partials, just re-located them.